data/engineering_ownership.yml

```diff
@@ -119,6 +119,14 @@ web_app:
   product_team: frontend_platform
   ownership_model: Owner
 
+resource_esimtator:
```

Comment:
@jyavorska I was just driving by and there's a typo here :)

Reply:
Thank you!

Reply:
Fixed via https://github.com/sourcegraph/handbook/pull/2630